server/bin/pbench-tarball-stats.py

```diff
@@ -125,17 +153,24 @@ def transform_buckets(buckets: dict):
 #     .ctrls      - dictionary of controllers with bad tar balls
 #   by_year       - dictionary mapping counts of tar balls for each given
 #                   year generated, with mappings per satellite server
-#   by_month      - dictionary mapping counts of tar balls for each given
-#                   month generated, with mappings per satellite server
+#   by_month      - same as by_year, just "by month"
+#   by_week       - list of dictionaries of weekly mappings, similar to by_year
+#                   and by_month but with data for the weeks of the most recent
+#                   two months, where we use the ISO week definition which
+#                   starts on a Monday and ends on a Sunday.
 #
-report_tmpl = """Summary Statistics for Tar Balls (external data only):
+# Each partial year, month, and week are marked with a capital 'P' to indicate
+# it as such, where "partial" is determined by considering the date the report
+# was created.
```

Comment:
I guess people reading the report regularly will know what "P" means  after a while, but I don't see any  explanation of it in the report itself. That will likely be  a head-scratcher for people who don't read the report regularly or see it for the first time.